index.bs

```diff
@@ -1993,10 +1993,10 @@ Charter Review and Approval</h3>
 New Charters and Major Changes</h4>
 
 	Any new [=charter=] (including re-chartering of existing [=chartered groups=])
-	and most substantive changes to an already-approved [=charter=]
+	and any change that is not a [=minor change=] (per [[#charter-minor-changes]]) to an already-approved [=charter=]
 	of a [=Working Group=] or [=Interest Group=]
 	<em class=rfc2119>must</em> be approved by a [=W3C Decision=] following an [=AC Review=] of that [=charter=].
-	(See [[#charter-minor-changes]] for minor changes that can be approved by a [=Team Decision=].)
+	Moditications to a [=charter=] <em class=rfc2119>should</em> have the [=consensus=] of the group.
```

Comment:
typo

```suggestion
	Modifications to a [=charter=] <em class=rfc2119>should</em> have the [=consensus=] of the group.
```

Reply:
But also, _modifications to ..._ reads strangely to me, though I know what is intended. Seems to me that _changes to ..._ is a more easily understood construct, even though they should mean the same thing.